tests/bluetooth/bsim_bt/bsim_test_audio/test_scripts/vcs.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2020-2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+SIMULATION_ID="vcs_vcp"
+VERBOSITY_LEVEL=2
+PROCESS_IDS=""; EXIT_CODE=0
+
+function Execute(){
+  if [ ! -f $1 ]; then
+    echo -e "  \e[91m`pwd`/`basename $1` cannot be found (did you forget to\
+ compile it?)\e[39m"
+    exit 1
+  fi
+  timeout 20 $@ & PROCESS_IDS="$PROCESS_IDS $!"
+}
+
+: "${BSIM_OUT_PATH:?BSIM_OUT_PATH must be defined}"
+
+#Give a default value to BOARD if it does not have one yet:
+BOARD="${BOARD:-nrf52_bsim}"
+
+cd ${BSIM_OUT_PATH}/bin
+
+printf "\n\n======== Running VCS standalone (API) test =========\n\n"
```

Comment:
Out of curiosity more than anything:
Why do you have 2 tests in the same script? (having them as 2 separate scripts allows running them in parallel, and would save you the need for the verbosity)

Reply:
Mostly due to my local development process I suppose: Easier to run just a single script to test the entire VCS. 

I could split it into two scripts but then I'd probably add a third script to run both of those anyhow :D

Reply:
aham :)
Maybe you'd want to use as 3rd script the one that is used in CI:
RESULTS_FILE=${ZEPHYR_BASE}/myresults.xml SEARCH_PATH=tests/bluetooth/bsim_bt/bsim_test_audio/ ./run_parallel.sh

Reply:
If I'm just testing e.g. VCS locally, then I have no intention of building all and running all tests (which does take quite a while compared), so I probably wouldn't do that :D